Controller/SearchController.php

```diff
@@ -77,4 +81,63 @@ public function index(Request $request, ProcessRepository $processRepository, Pa
             ]
         );
     }
+
+    /**
+     * @Route("/external", name="search_external")
+     */
+    public function external(Request $request, ProcessRepository $processRepository, PaginatorInterface $paginator)
+    {
+        $search = $request->query->get('search');
+
+        $qb = $processRepository->createQueryBuilder('e');
+        $qb->leftJoin('e.client', 'client');
+        $qb->addSelect('client');
+        $qb->leftJoin('e.caseWorker', 'caseWorker');
+        $qb->addSelect('partial caseWorker.{id,username}');
+        $qb->where('e.caseNumber = :search');
+        $qb->orWhere('e.clientCPR = :search');
+        $qb->orWhere('client.telephone = :search');
+        $qb->orWhere('client.address = :search');
+        $qb->orWhere(
+            $qb->expr()->concat(
+                $qb->expr()->concat('client.firstName', $qb->expr()->literal(' ')),
+                'client.lastName'
+            ).'= :search'
+        );
+        $qb->setParameter(':search', $search);
+
+        // Add sortable fields.
+        $qb->leftJoin('e.channel', 'channel');
+        $qb->addSelect('partial channel.{id,name}');
+
+        $qb->leftJoin('e.service', 'service');
+        $qb->addSelect('partial service.{id,name}');
+
+        $qb->leftJoin('e.processType', 'processType');
+        $qb->addSelect('partial processType.{id,name}');
+
+        $qb->leftJoin('e.processStatus', 'processStatus');
+        $qb->addSelect('partial processStatus.{id,name}');
+
+        $query = $qb->getQuery();
+
+        $pagination = $paginator->paginate(
+            $query,
+            $request->query->get('page', 1),
+            50
+        );
+        $pagination->setCustomParameters([
+            'align' => 'center',
+            'size' => 'small',
+            'style' => 'bottom',
+        ]);
```

Comment:
The code for performing the search is (apart form a few differences – see below) duplicated in the controller. Consider moving it to a service that can handle searches (perhaps depending on user's roles).

Search differences
* `LIKE` vs. `=` – by design or by typo?
* one searches in `caseWorker.username`; the other in `client.name` – by design?